docs_src/general/Definitions.md

```diff
@@ -1,6 +1,9 @@
 # Definitions
 The following glossary provides terms used in EdgeX Foundry.  The definition are based on how EdgeX and its community use the term versus any strict technical or industry definition.
 
+## Actuate
```

Comment:
it wont let me add a comment for another section of this document, but under the "Containerized" section, can you fix the "infrastrucutre" misspelling.